stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/QueryChannelsSpec.kt

```diff
@@ -1,11 +1,12 @@
 package io.getstream.chat.android.offline.querychannels
 
 import io.getstream.chat.android.client.api.models.FilterObject
+import io.getstream.chat.android.client.api.models.QuerySort
+import io.getstream.chat.android.client.models.Channel
 
 internal data class QueryChannelsSpec(
     val filter: FilterObject,
-    var cids: Set<String> = emptySet()
+    val querySort: QuerySort<Channel>
 ) {
-    val id: String
-        get() = filter.hashCode().toString()
+    var cids: Set<String> = emptySet()
```

Comment:
Why are we excluding it from the constructor?

Reply:
Kotlin compare data classes by fields from constructor, fields from the body of class aren't considered when equals and hashCode are running. That was my motivation move mutable cids from constructor to body